source/stylesheets/site.css.scss

```diff
@@ -1,24 +1,3 @@
-body {
-  background-color: #fbc547;
-  color: #333;
-  font-family: -apple-system, BlinkMacSystemFont, "Avenir Next", "Avenir",
-    "Segoe UI", "Lucida Grande", "Helvetica Neue", "Helvetica", "Fira Sans",
-    "Roboto", "Noto", "Droid Sans", "Cantarell", "Oxygen", "Ubuntu",
-    "Franklin Gothic Medium", "Century Gothic", "Liberation Sans", sans-serif;
-  padding: 18vh 1rem;
-  text-align: center;
-}
-
-a {
-  color: rgba(#000, 0.7);
-
-  &:focus,
-  &:hover {
-    color: rgba(#000, 0.6);
-  }
-}
-
-.middleman-logo {
-  margin-bottom: 1rem;
-  width: 10rem;
-}
+@import "base/global-variables";
```

Comment:
I'd change this partial's name to `_variables.scss`. If I'm another developer looking at this codebase for the first time, this makes me think that you have variables tucked away in other locations that I might have to look out for.

I do think it's a great call to carve out a space to contain these vars, regardless of name!